xcm/xcm-builder/src/location_conversion.rs

```diff
@@ -285,31 +364,24 @@ pub struct GlobalConsensusParachainConvertsFor<UniversalLocation, AccountId>(
 	PhantomData<(UniversalLocation, AccountId)>,
 );
 impl<UniversalLocation: Get<InteriorMultiLocation>, AccountId: From<[u8; 32]> + Clone>
-	Convert<MultiLocation, AccountId>
-	for GlobalConsensusParachainConvertsFor<UniversalLocation, AccountId>
+	ConvertLocation<AccountId> for GlobalConsensusParachainConvertsFor<UniversalLocation, AccountId>
 {
-	fn convert_ref(location: impl Borrow<MultiLocation>) -> Result<AccountId, ()> {
+	fn convert_location(location: &MultiLocation) -> Option<AccountId> {
 		let universal_source = UniversalLocation::get();
 		log::trace!(
 			target: "xcm::location_conversion",
 			"GlobalConsensusParachainConvertsFor universal_source: {:?}, location: {:?}",
-			universal_source, location.borrow(),
+			universal_source, location,
 		);
-		let devolved = ensure_is_remote(universal_source, *location.borrow()).map_err(|_| ())?;
+		let devolved = ensure_is_remote(universal_source, location.clone()).ok()?;
```

Comment:
using `clone` on `MultiLocation` which implements `Copy`
```suggestion
		let devolved = ensure_is_remote(universal_source, *location).ok()?;
```